snapcraft/tests/test_commands_init.py

```diff
@@ -37,6 +37,16 @@ def test_init_must_write_snapcraft_yaml(self):
         fake_logger = fixtures.FakeLogger(level=logging.INFO)
         self.useFixture(fake_logger)
 
+        expected_yaml = """name: # the name of the snap
+version: # the version of the snap
+summary: # 79 char long summary
+description: # a longer description for the snap
+confinement: devmode # devmode means the snap doesn't support confinement"""
```

Comment:
It doesn't presupposes, but it implies. That's a common trick to provide more information in less space. The latter information you mention is not provided in the original version or the suggested one, and I think that's okay. We can't document it all in a single line.